src/main/java/org/sagebionetworks/bridge/spring/controllers/StudyController.java

```diff
@@ -159,16 +165,37 @@ public String getAllStudies(@RequestParam(required = false) String format,
             Collections.sort(activeStudiesSummary, STUDY_COMPARATOR);
             ResourceList<Study> summaries = new ResourceList<Study>(activeStudiesSummary)
                     .withRequestParam("summary", true);
-            return Study.STUDY_LIST_WRITER.writeValueAsString(summaries);
+            return STUDY_LIST_WRITER.writeValueAsString(summaries);  
         }
         UserSession session = getAuthenticatedSession(ADMIN);
-        
         verifyCrossStudyAdmin(session.getId(), "Study admin cannot access all studies");
 
         // otherwise, return all studies including deactivated ones
         return BridgeObjectMapper.get().writeValueAsString(
                 new ResourceList<>(studies).withRequestParam("summary", false));
     }
+    
+    @GetMapping(path="/v3/studies/memberships", produces={APPLICATION_JSON_UTF8_VALUE})
+    public String getStudyMemberships() throws Exception {   
+        UserSession session = getAuthenticatedSession();
+        
+        if (session.getParticipant().getRoles().isEmpty()) {
+            throw new UnauthorizedException(STUDY_ACCESS_EXCEPTION_MSG);
+        }
+        List<String> studyIds = accountDao.getStudyIdsForUser(session.getParticipant().getSynapseUserId());
```

Comment:
Similarly, this should also go through the service.

Reply:
Filed a JIRA to refactor out a service class: https://sagebionetworks.jira.com/browse/BRIDGE-2636